.github/workflows/composer-lint.yml

```diff
@@ -0,0 +1,20 @@
+name: "Composer Lint"
+
+on:
+  pull_request:
+    branches:
+      - "*.x"
+    paths:
+      - "composer.json"
+  push:
+    branches:
+      - "*.x"
+    paths:
+      - "composer.json"
+
+jobs:
+  composer-lint:
+    name: "Composer Lint"
+    uses: "doctrine/.github/.github/workflows/composer-lint.yml@2.1.0"
+    with:
+      php-version: "8.1"
```

Comment:
```suggestion
```

Isn't that the default?

Reply:
Oh right, I copy/pasted from another repo.